js/renovation/ui/common/widget.tsx

```diff
@@ -420,9 +420,11 @@ export class Widget extends JSXComponent(WidgetProps) {
     } = this.props;
 
     const accessKey = focusStateEnabled && !disabled && this.props.accessKey;
+
     return {
-      ...extend({}, this.restAttributes, accessKey && { accessKey }) as Record<string, string>,
+      ...extend({}, accessKey && { accessKey }) as Record<string, string>,
       ...getAria({ ...aria, disabled, hidden: !visible }),
+      ...this.restAttributes,
```

Comment:
seems like we should cast it to `Record<string, stirng>`.
I'm also not sure that `restAttributes` can not be `undefined` for example. Maybe it's bette to write this way:
`...extend({}, this.restAttributes)  as Record<string, string>`

Reply:
good remark. I am not sure too https://github.com/DevExpress/DevExtreme/pull/22673/commits/deeb89db3457cca422513881aeded6e459ca0ad3